### PR TITLE
add my link and bio

### PR DESCRIPTION
**Pull request by arefathi**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1818"></a>

### CODE REVIEW

File: public/data/arefathi.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  ],
+  "milestones": [
+    {
+      "title": "",
+      "date": "",
+      "icon": "",
+      "color": "",
+      "description": "",
+      "url": ""
+    }
```
Comment (Panquesito7) [inline public/data/arefathi.json:26]: I recommend removing these fields if you're not going to use them.

File: public/data/arefathi.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+      "url": "https://github.com/arefathi",
+      "icon": "github"
+    },
+    {
+      "name": "",
+      "url": "",
+      "icon": ""
+    }
+  ],
+  "milestones": [
```
Comment (kumar-rocks) [inline public/data/arefathi.json:18]: If you are not adding anything in milestones section, then you can remove this section.

File: public/data/arefathi.json (hunk 1 above)
Comment (imlakshay08) [inline public/data/arefathi.json:26]: You need to remove these fields if you are not using them.

**Review by imlakshay08 — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks Good to me! 🎉

**Pull request merged**